### ISSUE TITLE
[Question] An error: 'LazySupervisedDataset' object has no attribute 'map'.

### ISSUE THREAD

**Issue by WentaoTan** [yihedeng9/STIC#3]
### Question

![1721895945795](https://github.com/user-attachments/assets/6d6ed38b-b4d5-4483-86bf-3e4ed7c85ee0)
When I downloaded the corresponding data and ran run/dpo_finetune.sh, this error occurred.

**Comment by WentaoTan** [comment]
Yes, I solved it. But it's been so many months that I've forgotten how I solved it. 
Can you try this version of trl to see if it can help you? trl==0.9.6

**Comment by WentaoTan** [comment]
Or refer to this stic/dpo_trainer.py for training. Sorry brother, I can't remember

**Comment by ZihaoZheng98** [comment]
> Or refer to this stic/dpo_trainer.py for training. Sorry brother, I can't remember

Wow, thanks! That's helpful. Thank you bro.
